tests/kola/misc/test-status.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash     
+
+# Simple sanity checks for systemd unit status updates.
+
+set -xeuo pipefail
+
+. ${KOLA_EXT_DATA}/libtest.sh
+
+cd $(mktemp -d)
+
+systemctl stop zincati.service
+echo "updates.enabled = false" > /etc/zincati/config.d/disable-in-test.toml
+systemctl start zincati.service
+systemctl status zincati.service | grep 'Status:' > zincati_disabled_status.txt
+assert_file_has_content zincati_disabled_status.txt \
+'initialization complete, auto-updates logic disabled by configuration'
+ok "status show initialization"
+
+systemctl stop zincati.service
+rm /etc/zincati/config.d/*
+echo "updates.enabled = true" > /etc/zincati/config.d/disable-in-test.toml
+systemctl start zincati.service
+systemctl status zincati.service | grep 'Status:' > zincati_polling_status.txt
+assert_file_has_content zincati_polling_status.txt 'periodically polling for updates'
+ok "status show polling"
+# Give Zincati some time to check for updates.
+sleep 30
```

Comment:
In order to reduce the test-run time, you may consider replacing this with a while loop with a more granular sleep, and assert on the file content once out of the loop.